modules/integration/src/test/java/io/ballerina/messaging/broker/integration/standalone/cli/InitCmdTest.java

```diff
@@ -118,4 +119,26 @@ public void testInitCmdWithParam() {
         Assert.assertEquals(configuration.getPassword(), "admin123", "password is invalid in the configuration file");
     }
 
+
+    @Test(description = "test configuration 'without password' invoked without providing a command password")
+    public void testNoPasswordParam() {
+        System.setProperty(TestConstants.CLI_CONFIG_SYSTEM_PROPERTY, InitCmdTest.TEMP_CONFIG_DIR + "cli-config3.yaml");
```

Comment:
Shall we reset the system property after the test  since this can affect the other tests?

Reply:
Resetting is done at the after class method in the parent class

Reply:
updated